clib/cUnix.mli

```diff
@@ -11,34 +11,38 @@
 (** {5 System utilities} *)
 
 type physical_path = string
-type load_path = physical_path list
+type physical_dir_path = string
+type load_path = physical_dir_path list
```

Comment:
Not a big deal for this PR (so not a request for changes), however I find these alias to be often a usability issue as they leak into error messages of type string.

IMHO we should discuss if we need to revisit the practice of adding these aliases, when they are convertible to other types.